docs/Context-targeting.md

```diff
@@ -51,7 +51,32 @@ Fliplet adds some custom tests to Modernizr to help detect and target the follow
 | Tablet `width >= 640px` | `tablet` |
 | iPhone X | `iphonex` |
 
-## Classes to show/hide content
+---
+
+## CSS Class to show content in Fliplet Studio while in edit mode
+
+Use the `.visible-interact` class to show your content only when is being seen in Fliplet Studio while in edit mode. Anywhere else (e.g. preview mode, web apps, iOS and Android apps) the elements will be hid.
```

Comment:
```suggestion
Use the `.visible-interact` class to show your content only when is being seen in Fliplet Studio while in edit mode. Otherwise, the elements will be hidden (e.g. preview mode, web apps, iOS and Android apps).
```